### ISSUE TITLE
I can not use FCM.presentLocalNotification on iOS.

### ISSUE THREAD

**Issue by MechanicKim** [evollu/react-native-fcm#218]
Please include following information for better support

What verion of RN and react-native-fcm are you running?
RN : 0.38
react-native-fcm : 2.3.2

What device are you using? (e.g iOS9 emulator, Android 6 device)?
iOS 10.1 emulator

Is your app running in foreground, background or not running?
foreground

Advanced:
Have you tried adding break point in native handlers to see if message gets delivered?
can share a project with issue?
I can share a captured image
![simulator screen shot 2016 12 6 10 50 36](https://cloud.githubusercontent.com/assets/12149783/20909998/37a10fc2-bba2-11e6-9a03-b8199ee5e252.png)

**Comment by MechanicKim** [comment]
I add error msg

Exception 'Invalid parameter not satisfying: identifier != nil' was thrown while invoking presentLocalNotification on target RNFIRMessaging with params (
        {
        body = "New message, check it out!";
        "local_notification" = 1;
        priority = high;
        "show_in_foreground" = 1;
        sound = default;
        tag = CHAT;
        title = Chat;
    },
    2,
    3
)

**Comment by C-call** [comment]
+1

**Comment by MechanicKim** [comment]
I found a solution.
@evollu I have a question.
In RNFIRMessaging.m file, why did you write line 245 ~ 254?
When I removed these lines and left line 255 ~ 257, Local notification worked well!

Could you explain about it?

**Comment by sibelius** [comment]
@evollu I think this is a regression on ios 10.1

**Comment by sibelius** [comment]
@evollu I can confirm the behavior described by @MechanicKim 

@MechanicKim if u remove the lines 255~257 it will show the LocalNotification twice

but with the current code it won't show any

**Comment by yannickoo** [comment]
Same here :(

**Comment by yannickoo** [comment]
I can confirm this 😞

**Comment by evollu** [comment]
for the original issue, adding `id` property to the payload should fix the problem

**Comment by sibelius** [comment]
the new way is not working in my app on iOS 10.1

however, the code below works fine

```js
UILocalNotification* notif = [RCTConvert UILocalNotification:data];
    [RCTSharedApplication() presentLocalNotificationNow:notif];
    resolve(nil);
```

**Comment by evollu** [comment]
it is a bug. Id now is required.
fixed in 2.5.2

**Comment by sibelius** [comment]
@evollu u can close this issue

let's move to this one: https://github.com/evollu/react-native-fcm/issues/226
